src/lib.rs

```diff
@@ -84,6 +131,7 @@ mod line;
 pub use line::*;
 
 mod lookup;
+pub use lookup::{DebugLookup, LookupEntryIter};
```

Comment:
Yeah. I added them because it felt weird not having them when browsing the rustdocs, but then I also fixed up the docs for the type aliases to be better, so I don't think its necessary.